#!/usr/bin/env python
PACKAGE = "car_interface"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("steer_trim", int_t, 0, "Steer trim value", 0, -1000, 1000)
gen.add("throttle_trim", int_t, 0, "Throttle trim value", 0, -1000, 1000)

exit(gen.generate(PACKAGE, "car_interface", "Trim"))

